lib/galaxy/job_metrics/instrumenters/cgroup.py

```diff
@@ -72,64 +73,49 @@ def job_properties(self, job_id, job_directory):
         return metrics
 
     def __record_cgroup_cpu_usage(self, job_directory):
-        return """if [ `command -v cgget` ] && [ -e /proc/$$/cgroup ]; then cat /proc/$$/cgroup | awk -F':' '$2=="cpuacct,cpu"{{print $2":"$3}}' | xargs -I{{}} cgget -g {{}} > {metrics} ; else echo "" > {metrics}; fi""".format(metrics=self.__cgroup_metrics_file(job_directory))
+        # comounted cgroups (which cpu and cpuacct are on the supported Linux distros) can appear in any order (cpu,cpuacct or cpuacct,cpu)
+        return """if [ -e "/proc/$$/cgroup" -a -d "{cgroup_mount}" ]; then cgroup_path=$(cat "/proc/$$/cgroup" | awk -F':' '($2=="cpuacct,cpu")||($2=="cpu,cpuacct"){{print $3}}'); for f in {cgroup_mount}/{{cpu\,cpuacct,cpuacct\,cpu}}$cgroup_path/{{cpu,cpuacct}}.*; do if [ -f "$f" ]; then echo "__$(basename $f)__" >> {metrics}; cat "$f" >> {metrics} 2>/dev/null; fi; done; fi""".format(metrics=self.__cgroup_metrics_file(job_directory), cgroup_mount=self.cgroup_mount)  # noqa: W605
```

Comment:
what about just replacing the newlines for more readable code here?

```suggestion
        return """
        	if [ -e "/proc/$$/cgroup" -a -d "{cgroup_mount}" ]; then
        		cgroup_path=$(cat "/proc/$$/cgroup" | awk -F':' '($2=="cpuacct,cpu")||($2=="cpu,cpuacct"){{print $3}}'); 
        		for f in {cgroup_mount}/{{cpu\,cpuacct,cpuacct\,cpu}}$cgroup_path/{{cpu,cpuacct}}.*; do
        			if [ -f "$f" ]; then
        				echo "__$(basename $f)__" >> {metrics};
        				cat "$f" >> {metrics} 2>/dev/null;
        			fi; 
        		done; 
        	fi""".replace('\n', '').format(metrics=self.__cgroup_metrics_file(job_directory), cgroup_mount=self.cgroup_mount)  # noqa: W605
```

Reply:
I honestly don't know why we tried to minify these for the job script. @jmchilton do you know?

Reply:
Oh I didn't notice your `.replace('\n', '')` in there which is what I ended up doing anyway in my version lol.